### PR TITLE
Save ccb to file py example

### PR DESCRIPTION
**Pull request by zfeng10**

### CODE REVIEW

File: bindings/python/examples/saveCCBToFile/saveCCBToFile.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    print("k2: ",intrinsicParameters.k2)
+    print("k3: ",intrinsicParameters.k3)
+    print("k4: ",intrinsicParameters.k4)
+    print("k5: ",intrinsicParameters.k5)
+    print("k6: ",intrinsicParameters.k6)
+    print("p2: ",intrinsicParameters.p2)
+    print("p1: ",intrinsicParameters.p1)
+
+    timestamp =  time.strftime('%y%m%d%H%M')
+    status = camera1.setControl("saveModuleCCB",dir_path + ccb_prefix +  timestamp + '.ccb')
```
Comment (SeptimiuVana) [inline bindings/python/examples/saveCCBToFile/saveCCBToFile.py:111]: This function is no longer called with setControl, you need to use this function:
https://github.com/analogdevicesinc/ToF/blob/master/sdk/include/aditof/camera.h#L200

File: bindings/python/examples/saveCCBToFile/saveCCBToFile.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        # Create the directory
+        os.makedirs(dir_path)
+        print(f"The directory {dir_path} was created.")
+
+    status = system.getCameraList(cameras, ip)
+    print("system.getCameraList()", status)
+
+    camera1 = cameras[0]
+
+    status = camera1.setControl("initialization_config", config)
```
Comment (SeptimiuVana) [inline bindings/python/examples/saveCCBToFile/saveCCBToFile.py:77]: This one is deprecated as well, you need to parse the config file in the initialize function:
https://github.com/analogdevicesinc/ToF/blob/master/sdk/include/aditof/camera.h#L67

File: bindings/python/examples/saveCCBToFile/saveCCBToFile.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+if __name__ == "__main__":
+
+    if len(sys.argv) < 2  or sys.argv[1] == "--help" or sys.argv[1] == "-h" :
+        print("save_ccb.py usage:")
+        print("USB / Local connection: save_ccb.py <config>")
+        print("Network connection: save_ccb.py <ip> <config>")
+        exit(1)
+
+    system = tof.System()
```
Comment (andrestraker) [inline bindings/python/examples/saveCCBToFile/saveCCBToFile.py:48]: Better suited after the parameters have been validated and directory created. So just before **status = system.getCameraList(cameras, ip)**.

File: bindings/python/examples/saveCCBToFile/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# First frame Example
+
+### Overview
+This example saves the hardwares's CCB to a file
```
Comment (andrestraker) [inline bindings/python/examples/saveCCBToFile/readme.md:4]: Change

**This example saves the hardwares's CCB to a file**

to

**This tool saves the CCB of the imager's module to a file.**

**Comment by andrestraker** [comment]
@zfeng10  and @AAbacan2  - can this be moved from the examples to the tools folder? What is the impact of doing so wrt the Python bindings?

**Comment by dNechita** [comment]
> @zfeng10 and @AAbacan2 - can this be moved from the examples to the tools folder? What is the impact of doing so wrt the Python bindings? If it is moved, the Readme.md file needs to be updated such that the user is aware there is a dependency on the Python bindings.

Moving to tools might complicate cmake, but is do-able. Would it be sufficient to create a tools dir next to examples dir? (ToF/bindings/python/tool)

File: bindings/python/aditofpython.cpp [hunk 5]
```diff
@@ -247,6 +247,8 @@ PYBIND11_MODULE(aditofpython, m) {
              py::arg("value"))
         .def("getControl", &aditof::Camera::getControl, py::arg("control"),
              py::arg("value"))
+        .def("saveModuleCCB", &aditof::Camera::saveModuleCCB,
+             py::arg("filepath"))
```
Comment (dNechita) [inline bindings/python/aditofpython.cpp:251]: Recently these changes have reached master. So you can remove them to avoid duplication.

**Comment by dNechita** [comment]
I am going to squash all these commits since there are 2 merge commits. The squash is because we want to have a linear commit history.

**Comment by andrestraker** [comment]
> > @zfeng10 and @AAbacan2 - can this be moved from the examples to the tools folder? What is the impact of doing so wrt the Python bindings? If it is moved, the Readme.md file needs to be updated such that the user is aware there is a dependency on the Python bindings.
> 
> Moving to tools might complicate cmake, but is do-able. Would it be sufficient to create a tools dir next to examples dir? (ToF/bindings/python/tool)

Ok. Let's leave it was is in the Python bindings example folder.

**Pull request merged**